web/content/docs/benchmarks/reactive-transport/wetland/Wetland.pandoc

```diff
@@ -0,0 +1,123 @@
+For wastewater pollutants entering the system, a time-dependent Dirichlet BC is defined at the influx point.
+Respective material properties, initial and boundary conditions are listed in Table 1--2.
+
+<!--![\n Network of microbiological reactions described by CWM1. Modified with permission from Samsó et al. (2013).](../Wetland_cwm1.png)-->
+
+
+-----------------------------------------
+
+|Parameter | Description | Value | Unit |
+|:-------- | :---------- | :---- | :--- |
+| Influent & effluent zone |||
+| $\phi$   | Porosity    | 0.38  | - |
+| $\kappa$ | Permeability | 1.0e-7 | m^2^ |
+| $S$   | Storage | 0.0     | - |
+| $a_L$ | long. Dispersion length | 0.45   | m |
+| Treatment zone |||
+| $\phi$   | Porosity    | 0.38  | - |
+| $\kappa$ | Permeability | 1.0e-8 | m^2^ |
+| $S$   | Storage | 0.0     | - |
+| $a_L$ | long. Dispersion length | 0.40   | m |
+| $\rho_w$ | Fluid density | 1.0e+3 | kg m^-3^ |
+| $\mu_w$ | Fluid viscosity | 1.0e-3 | Pa s |
+
+-----------------------------------------
+
+Table 1: Media, material and component properties
+
+
+-----------------------------------------
+
+| Parameter | Description | Value | Unit |
+|:--------- | :---------- | :---- | :--- |
+| $p(t=0)$  | Initial pressure | 8829  | Pa |
+| $g_{N,in}^p$ | Water influx | 5.555e-3 | kg s^-1^ |
+| $g_{D,out}^p$ | Pressure at outlet | 8829 | Pa |
+| $c_{pollutants}(t=0)$  | Initial solute concentrations | 1.0e-4     | g kg^-1^~water~ |
+| $c_{bacteria}(t=0)$  | Initial bacteria concentrations | 1.0e-3  | g kg^-1^~water~ |
+| $c_{\text{H}^+}(t=0)$  | Initial concentration of $\text{H}^+$ | 1.0e-7     |  mol kg^-1^~water~ |
+|  $g_{D,in}^{pollutants_c}$ | Influent solute concentrations |  $f(t)$ | g kg^-1^~water~ |
+|  $g_{D,in}^{\text{H}^+}$ | Influent concentration of $\text{H}^+$ |  1.0e-7 | mol kg^-1^~water~ |
+
+-----------------------------------------
+
+Table 2: Initial and boundary conditions
+
+
+-----------------------------------------
+
+
+## Results
+
+Initially, influent wastewater pollutants spread through the model domain by advective--dispersive transport (Figure 3).
+The aeration increases dissolved oxygen concentration $Do$ in the entire domain.
+However, after XXX seconds, rapidly growing heterotrophic bacteria $X_H$ starts to consume available $Do$ and influent organic carbon ($S_A$, $S_F$).
+Initial growth of $X_H$ and the high concentrations of organic carbon sources at the influent region result in an oxygen depletion.
+This triggers the growth of anaerobic fermenting bacteria $X_{FB}$ at the front and pushes $X_H$ downstream were enough $Do$ is available.
+After organic carbon concentration profiles of $S_F$ and $S_A$ stabilized, autotrophic nitrifying bacteria $X_A$ starts to grow by oxidizing available ammonia nitrogen ($S_{NH}$) to oxidized nitrogen $S_{NO}$.
+After XX seconds (20 days) the wetland has reached a quasi steady--state.
```

Comment:
The storage S equals to zero. I would expect that the steady-state reached in the first time step. Is my understanding wrong?

Reply:
I will have to clarify that I meant the state of the microbial reaction network, not the hydrologic system. Of course, water flow reaches steady--state in the first time step.